index.test-d.ts

```diff
@@ -0,0 +1,22 @@
+import {expectType} from 'tsd-check';
+import del, {sync as delSync} from '.';
```

Comment:
Am I the only one that finds this ugly? Should the `sync` just be a property of the default export instead of a named export? Or should we name the named export `delSync` so it doesn't have to be renamed to that?

Reply:
I'm not a fan of default imports. They're always kinda ugly, everyone names the thing differently and the tooling has hard times determining what you actually want to import. It's much easier with named exports.